libvast/builtins/formats/json.cpp

```diff
@@ -773,15 +798,32 @@ class json_printer final : public plugin_printer {
 
   auto instantiate(type, operator_control_plane&) const
     -> caf::expected<std::unique_ptr<printer_instance>> override {
+    const auto compact = !!args_.compact_output;
+    auto style = default_style();
+    if (detail::getenv("NO_COLOR") || args_.monochrome_output)
+      style = no_style();
+    else if (args_.color_output)
+      style = jq_style();
```

Comment:
I think `NO_COLOR` usually forces the default to be non-colored. But when explicitly requested, we should still color the output.

Reply:
This is confirmed by https://no-color.org, so the check should move into the `default_style()` function.

One additional note from that site: `NO_COLOR=` should also not cause colors to be disabled by default, i.e., the empty string should be the same as not having the env variable set.

Reply:
Fixed in 07ffd457ca74c2f77761d1aec0cdc766f14243b6